src/common/Finisher.h

```diff
@@ -132,17 +134,18 @@ class Finisher {
 
   /// Construct an anonymous Finisher.
   /// Anonymous finishers do not log their queue length.
-  Finisher(CephContext *cct_) :
+  Finisher(CephContext *cct_, const char *tn) :
```

Comment:
This could be a problem if someone will use this constructor in a old way, thinking that uses named Finisher, but it will be anonymous with named thread.